controllers/ncnews.controllers.js

```diff
@@ -0,0 +1,9 @@
+const {selectTopics} = require('../models/ncnews.models')
+
+exports.getTopics = (req, res, next) =>{
+   
+    selectTopics().then((topics) =>{
+        res.status(200).send({topics})
+    })
+    .catch(next)
```

Comment:
Great to see you've remembered to add a catch block here, we're not expecting anything to go wrong but its great that you have added it in just incase of a DB failure.